index.html

```diff
@@ -560,15 +564,18 @@
         <div dfn-for="MediaTrackSettings">
           <p>
             The <dfn><code>focalLength</code></dfn> dictionary member
-            represents the <a>depth map</a>'s <a>focal length</a>.
+            represents the <a>depth map</a>'s <a>focal length</a>. It is a
+            <a>non-constrainable property</a>.
```

Comment:
Or just mirror any other constraint, like `frameRate` (a numerical one is probably closer to what you want). There's nothing special with facingMode in this regard. I.e. add to:
* MediaTrackSupportedConstraints
* MediaTrackCapabilities
* MediaTrackConstraintSet
* MediaTrackSettings

(These four places used to be a single dictionary, but people who are not me decided to split them up).